client/levels/menu.tscn

```diff
@@ -127,6 +135,15 @@ func _on_button_back_pressed() -> void:
 func _ready() -> void:
 	if not Engine.is_editor_hint():
 		hide()
+	else:
+		update_credits()
+
+func update_credits() -> void:
```

Comment:
this loads the file when the editor is open, **It doesn't update it on cli** which means the pipeline is wrong.